.github/workflows/dashboard-testnet.yml

```diff
@@ -102,6 +104,7 @@ jobs:
               @keep-network/keep-core@${{ steps.upstream-builds-query.outputs.keep-core-contracts-version }} \
               @keep-network/keep-ecdsa@${{ steps.upstream-builds-query.outputs.keep-ecdsa-contracts-version }} \
               @keep-network/tbtc@${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }} \
+              @keep-network/tbtc-v2@${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }} \
```

Comment:
Shouldn't be`steps.upstream-builds-query.outputs.tbtc-v2-contracts-version` instead of `...tbtc-contracts-version`?

Reply:
Good catch, fixed in 606353bfb00998df6073fee9c6a7719e5aa335cc.